custom/icds_reports/views.py

```diff
@@ -285,7 +285,9 @@ def get_settings(self, request, *args, **kwargs):
         domain = self.kwargs['domain']
         current_month = datetime(year, month, 1)
         prev_month = current_month - relativedelta(months=1)
-        location = request.GET.get('location_id', '')
+        location = request.GET.get('location_id')
+        if location == 'null' or location == 'undefined':
+            location = None
```

Comment:
What happens if the location is `None`? What locations are shown on the page?

Reply:
The whole country is then selected.